lib/ddtrace/configuration.rb

```diff
@@ -35,34 +40,14 @@ def configure(target = configuration, opts = {})
       :runtime_metrics,
       :tracer
 
+    def shutdown!
+      components.teardown!
```

Comment:
If `@components` is `nil` here, it think it would be best to avoid lazy loading a new `Components` object just to shut it down.

Reply:
Oooh, yes, good call.